db/fixtures/acts_as_taggable_on/taggings.yml

```
@@ -55,3 +55,28 @@ question14_tag_manga:
   taggable: question14 (Question)
   context: tags
   tag: manga
+
+page3_tag_shinjuku_rb:
+  taggable: page3 (Page)
+  context: tags
+  tag: shinjuku_rb
+page4_tag_designer:
```

```suggestion

page4_tag_designer:
```

他のfixtureでは空行を開けてるので統一した方がいいかもです〜